test/adapter/json_api/resource_meta_test.rb

```diff
@@ -61,6 +75,24 @@ def test_meta_object_resource_in_array
             }
             assert_equal(expected, hash)
           end
+
+          def test_meta_object_blank_omitted
+            hash = ActiveModel::SerializableResource.new(
+              @post,
+              serializer: MetaBlockPostBlankMetaSerializer,
+              adapter: :json_api
+            ).serializable_hash
+            refute hash[:data].has_key? :meta
```

Comment:
refute?! woah, look who's good at minitest. :-)


srsly, I didn't know this was a thing